app/utils/track.js

```diff
@@ -203,6 +203,9 @@ export function trackPageView(isInitialPageView, uaDimensions) {
     M.trackingQueue.push(window.trackQuantcastPageView);
     M.trackingQueue.push(window.trackComscorePageView);
     M.trackingQueue.push((isOptedIn) => {
```

Comment:
Why do we need to wait for the tracking queue? Maybe we could remove everything that requires user opt in from that request and avoid depending on tracking opt-in load time?

Otherwise the event will still depend on additional factors and not only on the page load.

Reply:
it's only for subsequent page views so it won't depend on any other factors. The queue should be flushed immediately.

Reply:
https://github.com/Wikia/mercury-shared/blob/master/src/tracking/tracking-queue.js#L9